app/services/search/jobseeker_profile_search.rb

```diff
@@ -1,17 +1,19 @@
 class Search::JobseekerProfileSearch
   attr_reader :filters
 
-  def initialize(filters)
+  def initialize(current_organisation:, filters:)
     @filters = filters
-    @current_organisation = filters[:current_organisation]
+    @current_organisation = current_organisation
   end
 
   def jobseeker_profiles # rubocop:disable Metrics/AbcSize
     scope = JobseekerProfile
-      .includes(:job_preferences)
-      .includes(:personal_details)
-      .active.not_hidden_from(current_organisation)
-      .where(job_preferences: { id: location_preferences_ids_matching_location_search })
+              .includes(:job_preferences)
+              .includes(:personal_details)
```

Comment:
This seems to be overridden by the bellow `.left_outer_joins(:personal_details)`
```suggestion
```